app/views/miq_ae_customization/_dialog_field_form_non_dynamic_options.html.haml

```diff
@@ -1,204 +0,0 @@
-        = _('Value Type')
-      .col-md-8
-        = select_tag('field_data_typ',
-                      options_for_select([[_("Integer"), "integer"], [_("String"), "string"]],
-                                          @edit[:field_data_typ]),
-                      "data-miq_sparkle_on" => true,
-                      :class    => "selectpicker")
-        :javascript
-          miqSelectPickerEvent('field_data_typ', "#{url}")
-    .form-group
-      %label.col-md-2.control-label
-        = _('Protected')
-      .col-md-8
-        = check_box_tag('field_protected', 'true',
-                        @edit[:field_protected],
-                        "data-miq_observe_checkbox" => {:url => url}.to_json)
-  .form-group
-    %label.col-md-2.control-label= _('Required')
-    .col-md-8
-      = check_box_tag('field_required', 'true', @edit[:field_required],
-                      "data-miq_observe_checkbox" => {:url => url}.to_json)
-
-  - if %w(DialogFieldTextBox DialogFieldTextAreaBox).include?(@edit[:field_typ])
-    .form-group
-      %label.col-md-2.control-label
-        = _('Validator Type')
-      .col-md-8
-        = select_tag('field_validator_type',
-                      options_for_select([[_("None"), nil], [_("Regular Expression"), 'regex']],
-                                          @edit[:field_validator_type]),
-                      "data-miq_sparkle_on" => true,
-                      :class    => "selectpicker")
-      :javascript
-        miqSelectPickerEvent('field_validator_type', "#{url}")
-    .form-group
-      %label.col-md-2.control-label
-        = _('Validator Rule')
-      .col-md-8
-        \/
-        = text_field_tag("field_validator_rule",
-                          @edit[:field_validator_rule],
-                          "data-miq_observe" => {:interval => '.5', :url => url}.to_json,
-                          :disabled          => @edit[:field_validator_type].blank?,
-                          :class => "form-control",
-                          :maxlength         => 250)
-        \/
-
-- elsif %w(DialogFieldDateControl DialogFieldDateTimeControl).include?(@edit[:field_typ])
-  .form-group
-    %label.col-md-2.control-label
-      = _('Show Past Dates')
-    .col-md-8
-      = check_box_tag("field_past_dates", "1",
-                      @edit[:field_past_dates],
-                      "data-miq_observe_checkbox" => {:url => url}.to_json)
-
-- elsif %w(DialogFieldDropDownList DialogFieldRadioButton).include?(@edit[:field_typ])
-  .form-group
-    %label.col-md-2.control-label
-      = _('Required')
-    .col-md-8
-      = check_box_tag('field_required', 'true', @edit[:field_required],
-                      "data-miq_observe_checkbox" => {:url => url}.to_json)
-
-  .form-group
-    %label.col-md-2.control-label
-      = _('Default Value')
-    .col-md-8
-      = select_tag("field_default_value",
-                   default_value_form_options(@edit[:field_values], @edit[:field_default_value]),
```

Comment:
`default_value_form_options` method can be removed

spec/helpers/application_helper/dialogs_spec.rb
```
646:  describe "#default_value_form_options" do
647:    let(:subject) { helper.default_value_form_options(field_values, field_default_value) }
```


app/helpers/application_helper/dialogs.rb
```
136:  def default_value_form_options(field_values, field_default_value)
```